v2/bigquery-to-bigtable/src/main/java/com/google/cloud/teleport/v2/templates/BigQueryToBigtable.java

```diff
@@ -118,6 +119,60 @@ public interface BigQueryToBigtableOptions extends PipelineOptions {
     String getBigtableWriteColumnFamily();
 
     void setBigtableWriteColumnFamily(String value);
+
+    @TemplateParameter.Text(
```

Comment:
`TemplateParameter.Text`/`String` being used for what could be `TemplateParameter.Integer`/`String`? I would vote to make it restrictive here as UI can do a better job at validating before gets to us / fails pipeline / etc.

Reply:
Done.